com.neatdecisions.Detwinner.json

```diff
@@ -0,0 +1,167 @@
+{
+  "app-id": "com.neatdecisions.Detwinner",
+  "runtime": "org.gnome.Sdk",
+  "runtime-version": "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "detwinner",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=host",
+    "--filesystem=xdg-config/gtk-3.0",
```

Comment:
Why it does need this one?

Reply:
Maybe use `--persist` instead?

Reply:
But I am not sure if --persist can be used in combination with host file system access.

Reply:
In fact without this permission the user's bookmarks won't be shown in `GtkFileChooser` and `GtkPlacesSidebar`, even with `--filesystem=host`. Is there any better way to fix it?